app/Scripts/GlobalUpgradeStats.gd

```diff
@@ -0,0 +1,15 @@
+tool
+extends Node
+
+var globalSelfHealth: float = 100 #health of boids and player
+var boidDamage: float = 20
+var playerStamina: float = 100
+var globalEnemyHealth: float = 100 # health of all enemies
+var globalEnemyDamage: float = 20
```

Comment:
I think we should be creating the Upgrade resource here instead right?

Reply:
Wdym? This is just because we needed some global script to hold everything and everyone can access it to make it easier to implement upgrades